src/main/java/com/atlassian/tutorial/macro/ColorPick.java

```diff
@@ -0,0 +1,43 @@
+package com.atlassian.tutorial.macro;
+
+import com.atlassian.confluence.content.render.xhtml.ConversionContext;
+import com.atlassian.confluence.macro.Macro;
+import com.atlassian.confluence.macro.MacroExecutionException;
+import com.atlassian.confluence.renderer.radeox.macros.MacroUtils;
+import com.atlassian.confluence.util.velocity.VelocityUtils;
+import com.atlassian.plugin.spring.scanner.annotation.imports.ComponentImport;
+import com.atlassian.webresource.api.assembler.PageBuilderService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.inject.Named;
+import java.util.Map;
+
+@Named
+public class ColorPick implements Macro {
+
+    private final PageBuilderService pageBuilderService;
+
+    private final static String name = "name";
```

Comment:
Constant  name should be written in capital letters  `` private final static String NAME``